### PR TITLE
[DROOLS-2073] Kie server API to evaluate multiple decisions by name/id

### PR DESCRIPTION
**Pull request by kurobako**
https://issues.jboss.org/browse/DROOLS-2073

### CODE REVIEW

**Comment by kurobako** [comment]
tests are fine, I think it's maven plugin complaining about changes in protected API (command objects)
json/jaxb/xstream representation of objects should remain the same though, can someone please verify that?

**Comment by sutaakar** [comment]
@kurobako This is findbugs failure. You can run build with findbugs check running "mvn clean install -Dfull"

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/dmn/DMNResultKS.java [hunk 1]
```diff
@@ -105,13 +106,13 @@ public void setModelName(String modelName) {
...
-    public String getDecisionName() {
-        return decisionName;
+    public List<String> getDecisionNames() {
+        return decisionNames;
     }
 
     
-    public void setDecisionName(String decisionName) {
-        this.decisionName = decisionName;
+    public void setDecisionNames(List<String> decisionName) {
```
Comment (sutaakar) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/dmn/DMNResultKS.java]: Findbugs revealed a bug here. decisionName should be renamed to decisionNames.

Comment (kurobako) [reply kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/dmn/DMNResultKS.java]: @sutaakar indeed. time to check my eyes :) fixed

**Comment by sutaakar** [comment]
@kurobako Now it complains about compilation failure.

**Comment by kurobako** [comment]
@sutaakar oops. should be good now

**Comment by kurobako** [comment]
[INFO] Error size is 0
[INFO] Total bugs: 2
[INFO] Impossible downcast of toArray() result to String[] in org.kie.server.services.dmn.ModelEvaluatorServiceBase.evaluateDecisions(String, String, String) [org.kie.server.services.dmn.ModelEvaluatorServiceBase] At ModelEvaluatorServiceBase.java:[line 149] BC_IMPOSSIBLE_DOWNCAST_OF_TOARRAY
[INFO] Impossible downcast of toArray() result to String[] in org.kie.server.services.dmn.ModelEvaluatorServiceBase.evaluateDecisions(String, String, String) [org.kie.server.services.dmn.ModelEvaluatorServiceBase] At ModelEvaluatorServiceBase.java:[line 152] BC_IMPOSSIBLE_DOWNCAST_OF_TOARRAY

gosh. okaaaay

**Comment by etirelli** [comment]
As point 1 is minor (toString() method), I will go ahead and merge it as is. @kurobako it would be good to clean it in another opportunity.

**Pull request merged**